packages/atomic/cypress/integration/icon.cypress.ts

```diff
@@ -54,4 +56,16 @@ describe('Icon Test Suites', () => {
     setupIcon(url);
     cy.get(IconSelectors.icon).should('not.have.attr', 'xss');
   });
+
+  describe('when the icon is invalid', () => {
```

Comment:
Remember that this isn't a test for `atomic-result-icon`. `atomic-result-icon` uses `atomic-icon`, and `atomic-icon` is used by several components in the UI (but will likely also be used on its own in result templates).